scripts/consts.ts

```diff
@@ -0,0 +1,24 @@
+import { Address } from 'viem'
+import { FilterFunction, NetworkId, Protocol } from './types'
+import { filterEvents as beefy } from './protocol-filters/beefy'
+
+export const NETWORK_ID_TO_REGISTRY_ADDRESS = {
```

Comment:
Since the registry address will always be the same across chains, can we simplify this to just be a single const?

Reply:
I split out these changes into https://github.com/mobilestack-xyz/funding-layer/pull/21. If we deploy to ETH mainnet at a later date will it still have the same address?

Reply:
Yep, any chains we deploy to in the future will have the same address.

Reply:
This object is used in a few other places, so I'm opting to leave it as is and fix it in a subsequent PR.